iphone/cli/hooks/frameworks.js

```diff
@@ -422,12 +429,12 @@ class InspectFrameworksTask extends IncrementalFileTask {
 	 * @access private
 	 */
 	inspectFrameworks(frameworkPaths) {
-		let metadataPromises = [];
-		let frameworkInspector = new FrameworkInspector(this._logger);
-		for (let frameworkPath of frameworkPaths) {
-			let metadataPromise = frameworkInspector.inspect(frameworkPath).then(frameworkInfo => {
+		const metadataPromises = [];
+		const frameworkInspector = new FrameworkInspector(this._logger);
+		for (const frameworkPath of frameworkPaths) {
+			const metadataPromise = frameworkInspector.inspect(frameworkPath).then(frameworkInfo => {
 				if (this._frameworks.has(frameworkInfo.name)) {
```

Comment:
<!--
  0 failure: 
  1 warning:  iphone/cli/hooks/...
  
  
  DangerID: danger-id-Danger;
  File: iphone/cli/hooks/frameworks.js;
  Line: 436;
-->

- :warning: iphone/cli/hooks/frameworks.js line 436 – Each then() should return a value or throw (promise/always-return)